include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/TypeStateDescriptions/OpenSSLEVPKDFCTXDescription.h

```diff
@@ -72,17 +72,18 @@ class OpenSSLEVPKDFCTXDescription : public TypeStateDescription {
 
   // std::map<std::pair<const llvm::Instruction *, const llvm::Value *>, int>
   //     requiredKDFState;
-  IDESolver<IDETypeStateAnalysisDomain> &kdfAnalysisResults;
+  IDESolver<IDETypeStateAnalysisDomain> &KDFAnalysisResults;
   static OpenSSLEVTKDFToken funcNameToToken(const std::string &F);
 
 public:
   OpenSSLEVPKDFCTXDescription(
-      IDESolver<IDETypeStateAnalysisDomain> &kdfAnalysisResults)
-      : kdfAnalysisResults(kdfAnalysisResults) {}
-  bool isFactoryFunction(const std::string &F) const override;
-  bool isConsumingFunction(const std::string &F) const override;
-  bool isAPIFunction(const std::string &F) const override;
-  TypeStateDescription::State
+      IDESolver<IDETypeStateAnalysisDomain> &KDFAnalysisResults)
+      : KDFAnalysisResults(KDFAnalysisResults) {}
+
+  [[nodiscard]] bool isFactoryFunction(const std::string &F) const override;
+  [[nodiscard]] bool isConsumingFunction(const std::string &F) const override;
+  [[nodiscard]] bool isAPIFunction(const std::string &F) const override;
```

Comment:
```suggestion
  [[nodiscard]] bool isFactoryFunction(const std::string &FuncName) const override;
  [[nodiscard]] bool isConsumingFunction(const std::string &FuncName) const override;
  [[nodiscard]] bool isAPIFunction(const std::string &FuncName) const override;
```

`F` is used in LLVM for functions, just putting a function name as `F` into a function is a bit confusing when reading the implementation.